src/i18n/types.ts

```diff
@@ -15,8 +15,18 @@ export enum AUDIFICATION {
   ACTIVE_DATUM,
 }
 
-export enum AUDIFICATION_PREFERENCE {
+export enum DATA_PREFERENCE {
   enabled = 0x300,
+  avgHits,
+  hitStdDev,
+  avgUsers,
+  userStdDev,
+  avgSessionsPerUser,
+  sessionsPerUserStdDev,
+}
+
+export enum AUDIFICATION_PREFERENCE {
+  enabled = 0x400,
```

Comment:
just a question, how is this value used?

Reply:
`I18nKey` is a union type of all these enums. These values are used to resolve the conflict when defining i18n text for these enums in the same object.

Reply:
Could you just add a comment describing what these values are for? Thanks!